math/dev/density_plotting.clj

```diff
@@ -1,14 +1,14 @@
 (ns density-plotting
```

Comment:
We can actually remove this entire namespace; It was an experiment and note really needed any more. This sort of thing tends to go in the analysis notebooks now: https://github.com/compdemocracy/analysis.